assets/src/stories-editor/blocks/amp-story-text/index.js

```diff
@@ -40,6 +40,10 @@ const schema = {
 	align: {
 		type: 'string',
 	},
+	isPasted: {
```

Comment:
Does including it here mean that this attribute will be stored in WP for every text block? Or is it still only an internal attribute in the frontend initialised to `false` for all blocks?

Just want to make sure we aren't polluting the database with unnecessary attributes.

Reply:
It doesn't seem to go to the backend at all, everything looks good here.